src/Sunburst.ts

```diff
@@ -559,14 +554,6 @@ export class Sunburst implements IVisual {
         data.total += newDataPointNode.value;
         newDataPointNode.children = [];
 
-        if (level === 1 && originParentNode.children.length > 0) {
```

Comment:
See: https://github.com/microsoft/powerbi-visuals-sunburst/pull/95#discussion_r1282516333. They don't do anything because the variables are never used.